src/server/app.js

```diff
@@ -36,6 +37,7 @@ app.use(express.static(path.resolve(__dirname, '../../', 'build')))
 app.get('/embed/:id', routeHelper.player)
 app.get('/play/:id', routeHelper.player)
 app.get('/oembed', cors(), oembedRoute)
+app.get('/mail/send', mailRoute.send)
```

Comment:
I know this is a WIP but can this be a `/post`?

Reply:
yeah it should.. this is just for testing and it'll be cleaned up once it's ready.